src/components/button-toggle/button-toggle.pw.tsx

```diff
@@ -0,0 +1,863 @@
+    await checkGoldenOutline(toggleButton3);
+  });
+
+  test("should render with the expected styling when opt out flag is false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ButtonToggleGroupComponent />);
+
+    const toggleButton1 = buttonToggleButton(page).nth(0);
+    const toggleButton2 = buttonToggleButton(page).nth(1);
+    const toggleButton3 = buttonToggleButton(page).nth(2);
+
+    await toggleButton1.focus();
+    await expect(toggleButton1).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+    await expect(toggleButton1).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+    await toggleButton2.focus();
+    await expect(toggleButton2).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+    await expect(toggleButton2).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+    await toggleButton3.focus();
+    await expect(toggleButton3).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+    await expect(toggleButton3).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+  });
+
+  test("has render with the expected border-radius styling on a single toggle button", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ButtonToggle>Foo</ButtonToggle>);
+
+    await expect(buttonToggleButton(page)).toHaveCSS("border-radius", "4px");
+  });
+
+  test("should render with the expected border-radius styling when the children have the grouped prop set", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ButtonToggleGroupComponentGroupedChildren />);
+
+    await expect(buttonToggleButton(page).nth(0)).toHaveCSS(
+      "border-radius",
+      "4px 0px 0px 4px"
+    );
+    await expect(buttonToggleButton(page).nth(1)).toHaveCSS(
+      "border-radius",
+      "0px"
+    );
+    await expect(buttonToggleButton(page).nth(2)).toHaveCSS(
+      "border-radius",
+      "0px 4px 4px 0px"
+    );
+  });
+
+  test("should render with the expected border-radius styling when children do not have grouped prop set", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ButtonToggleComponent />);
+
+    await expect(buttonToggleButton(page).nth(0)).toHaveCSS(
+      "border-radius",
+      "4px"
+    );
+    await expect(buttonToggleButton(page).nth(1)).toHaveCSS(
+      "border-radius",
+      "4px"
+    );
+    await expect(buttonToggleButton(page).nth(2)).toHaveCSS(
+      "border-radius",
+      "4px"
+    );
+  });
+});
+
+test.describe("Prop tests", () => {
+  test("should render with aria-label prop", async ({ mount, page }) => {
+    await mount(<ButtonToggleComponent aria-label="cypress-aria" />);
+
+    await expect(buttonToggleButton(page).first()).toHaveAttribute(
+      "aria-label",
+      "cypress-aria"
```

Comment:
Can the label be changed to something that doesn't reference cypress? It's not a blocker but it doesn't really make sense now.